addon/utils/resize-observer-waiter.js

```diff
@@ -0,0 +1,23 @@
+import { buildWaiter } from '@ember/test-waiters';
+
+const waiter = buildWaiter('yapp-scroll-view:resize-observer');
+
+export function invokeResizeCallback(callback) {
+  let token;
+
+  if (waiter && typeof waiter.beginAsync === 'function') {
+    token = waiter.beginAsync();
+  }
+
+  try {
+    return callback();
+  } finally {
+    if (token) {
+      waiter.endAsync(token);
+    }
+  }
+}
+
+export function _resizeObserverWaiterForTesting() {
```

Comment:
It seems weird to prefix with `_` as if it's private, but to export the function